modin/pandas/test/utils.py

```diff
@@ -737,6 +737,31 @@ def applyier(module, *args, **kwargs):
     )
 
 
+def eval_io_from_str(csv_str: str, unique_filename: str, **kwargs):
+    """Evaluate I/O operation outputs equality check by using `csv_str`
+    data passed as python str (csv test file will be created from `csv_str`).
+
+    Parameters
+    ----------
+    csv_str: str
+        Test data for storing to csv file.
+    unique_filename: str
+        csv file name.
+    """
+    try:
+        with open(unique_filename, "w") as f:
+            f.write(csv_str)
```

Comment:
This is should be `pytest.fixture`.

Or maybe you can use `io.StringIO`.

Reply:
What is reason we should use `pytest.fixture` here?
I'm not sure we have to perform in-memory operation instead of reading from disk for `read_csv` function. Do you think it is equivalent operations in this case?

Reply:
It is one of the best practices to carry out all the logic needed to prepare for a test. In this case, `pytest` invites us to use `fixtures`. https://docs.pytest.org/en/stable/fixture.html.

Pandas works well with buffers (StringIO), I don't know Modin behavior here.

Reply:
Did not pay attention that this is not a test.

Out of date.